NumLib/DOF/LocalToGlobalIndexMap.cpp

```diff
@@ -191,4 +191,24 @@ std::ostream& operator<<(std::ostream& os, LocalToGlobalIndexMap const& map)
 }
 #endif  // NDEBUG
 
+NumLib::LocalToGlobalIndexMap::RowColumnIndices getRowColumnIndices(
```

Comment:
Thanks for pointing this out, the changes concerning dof table are moved to a separat PR #1442.